lib/arbre/html/document.rb

```diff
@@ -30,7 +29,7 @@ def to_s
 
       def build_head
         @head = head do
-          meta :"http-equiv" => "Content-type", content: "text/html; charset=utf-8"
+          meta "http-equiv": "Content-type", content: "text/html; charset=utf-8"
```

Comment:
turned back to draft. not sure this is working

Reply:
Not sure that `document` is providing the correct output

```
> Arbre::HTML::Document.new
<!DOCTYPE html><html></html>
```

I would have expected

```html
<!DOCTYPE html>
<html>
  <head>
    <meta .../>
  </head>
  <body></body>
</html>
```

Reply:
Yes, I recall noticing that when working on another change on removing defaults. I don't believe it has worked correctly for awhile. You can try on the default branch to confirm but I'm confident this was occurring there already and unrelated to your changes so I'm fine with this as is.

Reply:
Confirmed broken on v1.7.0